### PR TITLE
add consistent loading buttons around the platform

### PR DESCRIPTION
**Pull request by denisdimitrov**
Adds the correct loading styles from the mock ups repo and uses that around the platform for consistent UI.

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1815

### CODE REVIEW

File: src/components/admin/Button/Button.tsx [hunk 1]
```diff
@@ -28,10 +28,12 @@ export const Button: React.FC<ButtonProps> = ({
...
     {...extraParams}
     type={type}
-    className={cn(TW.general, TW[variant], TW[borders])}
+    className={cn(TW.general, TW[variant], TW[borders], {
+      [TW.loading]: loading,
+    })}
     ref={forwardRef}
   >
-    {loading && <Loading />}
+    {loading && <Loading containerClassName="pr-4" />}
```
Comment (goran-peoski-work) [inline src/components/admin/Button/Button.tsx]: [OOS] There are two not mutually exclusive ways of dealing with issues in this:
1. use variance props not injected classes
2. use a facade that works for both admin and attendee sides (similar to https://github.com/sparkletown/sparkle/blob/redesign-2021/src/components/shared/NotFound.tsx )

File: src/components/admin/SpaceCreateForm/SpaceCreateForm.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ import { PortalList } from "components/molecules/PortalList";
 import { SubmitError } from "components/molecules/SubmitError";
 import { YourUrlDisplay } from "components/molecules/YourUrlDisplay";
 
-import { ButtonNG } from "components/atoms/ButtonNG";
```
Comment (goran-peoski-work) [inline src/components/admin/SpaceCreateForm/SpaceCreateForm.tsx:38]: Did we get to a point that will make it safe to just delete `ButtonNG`?

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/admin/SpaceCreateForm/SpaceCreateForm.tsx (hunk 2 above)
Comment (denisdimitrov) [reply src/components/admin/SpaceCreateForm/SpaceCreateForm.tsx:38]: Almost! Some of the Attendee components are still using the `NG` version of the button.

**Pull request merged**